cves/kernel/CVE-2021-38198.yml

```
@@ -153,10 +156,10 @@ unit_tested:
 
     For the fix_answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  code:
-  code_answer:
-  fix:
-  fix_answer:
+  code: false
+  code_answer: Original Code was not Unit Tested
+  fix: true
+  fix_answer: Unit Tests added, https://lore.kernel.org/kvm/20210603050537.19605-1-jiangshanlai@gmail.com/
```

```suggestion
  fix_answer: |
    Unit Tests added, https://lore.kernel.org/kvm/20210603050537.19605-1-jiangshanlai@gmail.com/
```

YAML doesn't like colons in one-line strings like that